From e34373926377ecb4ae98c499d8c73e4a9be04dfd Mon Sep 17 00:00:00 2001
From: Eric Timmons <etimmons@mit.edu>
Date: Thu, 22 Oct 2020 17:04:14 -0400
Subject: [PATCH] Remove -Wimplicit-fallthrough

---
 src/runtime/Config.x86-64-linux | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/runtime/Config.x86-64-linux b/src/runtime/Config.x86-64-linux
index 008e2af9a..3b8801d9e 100644
--- a/src/runtime/Config.x86-64-linux
+++ b/src/runtime/Config.x86-64-linux
@@ -44,7 +44,7 @@ ifdef LISP_FEATURE_LIBUNWIND_BACKTRACE
   OS_LIBS += -lunwind
 endif
 
-CFLAGS += -Wunused-parameter -Wimplicit-fallthrough -fno-omit-frame-pointer -momit-leaf-frame-pointer
+CFLAGS += -Wunused-parameter -fno-omit-frame-pointer -momit-leaf-frame-pointer
 DISABLE_PIE=no
 
 ifdef LISP_FEATURE_IMMOBILE_SPACE
-- 
2.28.0

